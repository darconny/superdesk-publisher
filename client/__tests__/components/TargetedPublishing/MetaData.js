import React from "react";
import MetaData from "../../../components/TargetedPublishing/MetaData";
import { render, fireEvent } from "@testing-library/react";

describe("TargetedPublishing/MetaData", () => {
  const item = {
    guid: "asdfasdf87876876"
  };

  it("renders correctly", () => {
    const { container } = render(
      <MetaData
        apiUrl="example.com/"
        apiHeader={{ Authrization: "Basic 1234567" }}
        item={item}
      />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
