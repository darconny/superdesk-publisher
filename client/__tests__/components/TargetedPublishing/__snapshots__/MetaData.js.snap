// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TargetedPublishing/MetaData renders correctly 1`] = `
<div
  class="toggle-box toggle-box--dark sp--dark-ui toggle-box--circle"
>
  <div
    class="toggle-box__header"
  >
    <div
      class="toggle-box__chevron"
    >
      <i
        class="icon-chevron-right-thin"
      />
    </div>
    <div
      class="toggle-box__label"
    >
      Meta data
    </div>
    <div
      class="toggle-box__line"
    />
  </div>
  <div
    class="toggle-box__content-wraper"
  >
    <div
      class="toggle-box__content"
    >
      <div
        class="sd-list-item-group sd-shadow--z1"
      >
        <div
          class="sd-list-item"
        >
          <div
            class="sd-list-item__border"
          />
          <div
            class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border"
          >
            <div
              class="sd-list-item__row"
            >
              <span
                class="sd-overflow-ellipsis"
              >
                Facebook
              </span>
            </div>
          </div>
          <div
            class="sd-list-item__action-menu"
          >
            <i
              class="icon-chevron-right-thin"
            />
          </div>
        </div>
        <div
          class="sd-list-item"
        >
          <div
            class="sd-list-item__border"
          />
          <div
            class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border"
          >
            <div
              class="sd-list-item__row"
            >
              <span
                class="sd-overflow-ellipsis"
              >
                Twitter
              </span>
            </div>
          </div>
          <div
            class="sd-list-item__action-menu"
          >
            <i
              class="icon-chevron-right-thin"
            />
          </div>
        </div>
        <div
          class="sd-list-item"
        >
          <div
            class="sd-list-item__border"
          />
          <div
            class="sd-list-item__column sd-list-item__column--grow sd-list-item__column--no-border"
          >
            <div
              class="sd-list-item__row"
            >
              <span
                class="sd-overflow-ellipsis"
              >
                SEO
              </span>
            </div>
          </div>
          <div
            class="sd-list-item__action-menu"
          >
            <i
              class="icon-chevron-right-thin"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
